app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb

```diff
@@ -261,7 +261,7 @@ def parse_resource_pool(object, kind, props)
                    cached_parent = cache.find(parent) if parent
                    parent_model = persister.vim_class_to_collection(parent).base_class_name
 
-                   "Default for #{Dictionary.gettext(parent_model, :type => :model, :notfound => :titleize)} #{cached_parent[:name]}"
+                   "Default for #{parent_model.titleize} #{cached_parent[:name]}"
```

Comment:
`Dictionary.gettext()` does return localized string, regardless of where it is called (backend or frontend) and in this particular case it could lead to strings partially in English, partially in some other locale.

I'm not entirely sure where the resulting string will be used, though if it's not meant to be propagated into UI in some way, this fix is OK.

Reply:
Though now I see in the CI that it shows strings like `Default for Ems Cluster DC0_C0`, which I'm not sure is desirable.

One way to solve this is to prevent `Dictionary.gettext()` from returning localized strings with `:translate => false` parameter:
```ruby
Dictionary.gettext(parent_model, :type => :model, :notfound => :titleize, :translate => false)
```

Reply:
This will be the name of the resource pool so it will be shown on the UI.  It sounds like the whole name should be translated then, let me update that.

Reply:
@agrare ping? Or if you'd like, I can take over and finish the PR.

Reply:
Hey sorry I haven't had time to get back to this, if you want to take it over that would be awesome